praetorian_cli/handlers/access.py

```
@@ -0,0 +1,207 @@
+                'organizations',
+                aws_access_key_id=access_key,
+                aws_secret_access_key=secret_key,
+                aws_session_token=session_token,
+                region_name='us-east-1'
+            )
+            paginator = org_client.get_paginator('list_accounts')
+            for page in paginator.paginate():
+                sub_accounts.extend(page['Accounts'])
+        except Exception:
+            # Not an org account — that's fine, just use the root
+            pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)